wodles/aws/aws_s3.py

```diff
@@ -2318,6 +2318,40 @@ def __init__(self, *args, **kwargs):
         self.service = 'elasticloadbalancing'
         AWSCustomBucket.__init__(self, *args, **kwargs)
 
+    def db_count_custom(self, aws_account_id):
+        try:
+            query_count_custom = self.db_connector.execute(
+                    self.sql_count_custom.format(
+                        table_name=self.db_table_name,
+                        bucket_path=self.bucket_path,
+                        aws_account_id=aws_account_id,
+                        retain_db_records=self.retain_db_records
+                        ))
+            return query_count_custom.fetchone()[0]
+        except Exception as e:
+            print("ERROR: Failed to execute DB cleanup - Path: {bucket_path}: {error_msg}".format(
+                bucket_path=self.bucket_path,
+                error_msg=e))
+            sys.exit(10)
```

Comment:
There is no need to create a new function for this. You should expand the existing `db_count_custom` function present in the AWS Custom bucket class to add the `aws_account_id` as an optional parameter.